compiler/benchmarks/cakeml_benchmarks/cakeml/Holmakefile

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
why blank line?
Also I think I'd like to see `INCLUDES =` first, for consistency